packages/react/src/file-upload/FileUpload.tsx

```diff
@@ -73,6 +75,7 @@ export const FileUpload = forwardRef<HTMLInputElement, FileUploadProps>(
 			disabled,
 			label,
 			hideOptionalLabel,
+			hideThumbnails,
```

Comment:
Can we default this to `false`? :) 

```suggestion
			hideThumbnails = false,
```